industrial_ci/src/docker.sh

```diff
@@ -162,7 +162,11 @@ function ici_build_default_docker_image() {
   ici_docker_build - <<EOF > /dev/null
 FROM ubuntu:$UBUNTU_OS_CODE_NAME
 
-RUN apt-get update -qq && apt-get -qq install --no-install-recommends -y wget ca-certificates sudo
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
```

Comment:
I have seen `DEBIAN_FRONTEND` with docker in many places and no `debconf`. I think it is easier to get what `DEBIAN_FRONTEND` does, in addtion it's faster at build time.